File: example/javalib/testing/7-test-quick/foo/test/src/FooTest2.java
package foo;

import org.junit.Test;
import static org.junit.Assert.*;

public class FooTest2 {
    @Test
    public void test2() {
        String name = "Bob";
        String greeted = Foo.greet2(name);
        assertEquals("Hi, " + name + "!", greeted);
    }
}

File: example/kotlinlib/testing/7-test-quick/foo/test/src/FooTest2.kt
package foo

import org.junit.jupiter.api.Test
import org.junit.jupiter.api.Assertions.*

class FooTest2 {
    @Test
    fun test2() {
        val name = "Bob"
        val greeted = Foo.greet2(name)
        assertEquals("Hi, $name!", greeted)
    }
}